toolchain/parser/parser.cpp

```diff
@@ -17,6 +17,18 @@
 
 namespace Carbon {
 
+static auto ToString(Parser::ParseContext parse_context) -> std::string {
```

Comment:
It looks like this is probably just for formatv; similar is done on line 51 with operator<<, does that approach work here too?

Can you please add comments explaining what this is for, especially if it isn't matching my guess?